common/lib/xmodule/xmodule/course_module.py

```diff
@@ -712,6 +712,12 @@ class CourseFields(object):
         scope=Scope.settings,
         default=""
     )
+    web_view_certificates = Boolean(
+        display_name=_("Web View Certificates"),
+        help=_("Enter true or false. If true, web view certificates are enabled for the course"),
```

Comment:
"If true, certificate web/HTML views are enabled for the course." 
(in keeping with the change in line 716?)